### ISSUE TITLE
[Bug]: Unreadable  text

### ISSUE THREAD

**Issue by kosorin** [FreeTubeApp/FreeTube#2782]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for open and closed issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

1. Play any video
2. See unreadable text

![221029-1100-33](https://user-images.githubusercontent.com/13920314/198823814-ffdd5363-3fa9-4096-8d41-355d63cbaab9.png)
![221029-1130-15](https://user-images.githubusercontent.com/13920314/198824316-5ade28cb-077a-46e3-97a6-74c5707f13d5.png)

### Expected Behavior

Readable text

### Issue Labels

accessibility issue

### FreeTube Version

v0.17.1

### Operating System Version

Linux 6.0.2-arch1-1

### Installation Method

AUR

### Primary API used

Local API

### Additional Information

I tried different theme settings, nothing helped.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi there, we just did a release. Is this still an issue for u in v0.18.0?

**Comment by kosorin** [comment]
Yes, it's still an issue

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @kosorin can you try a nightly build and check if u can still reproduce the bug? 
https://docs.freetubeapp.io/development/nightly-builds

https://github.com/FreeTubeApp/FreeTube/actions/runs/4045534050

**Comment by kosorin** [comment]
I tried `freetube_0.18.0-nightly-2606_amd64.AppImage.zip`, same result.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
We think that this might me a platform specific issue as @maplepy also uses Arch and had the same issue.

**Comment by maplepy** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc oops sorry about the other issue, couldn't find any issue with `text` that was relevant, I guess the issue was so long ago that I couldnt find any :/

I'm trying a fresh arch install and will tell you about it

**Comment by maplepy** [comment]
Just tried with a brand new install or Freetube on a brand new Arch and it's still broken. Maybe it has to do with a missing font ?

**Comment by maplepy** [comment]
@kosorin what DE or WM are you using ? and did you install any specific font?

**Comment by kosorin** [comment]
I use awesome WM. I tried freetube on an old laptop (also arch/awesome WM) and it's ok there. So maybe it's graphic driver issue?

On PC I have gtx 1060. That old laptop should have "Intel GMA HD 3000 (integrated) and AMD Radeon HD 6630M (dedicated)"

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Bot sure if i already asked this but did u try checking for errors in the console?

Command to open the console: `CTRL+SHIFT+i`

Make sure to open it right after u opened FT.

**Comment by maplepy** [comment]
This is what I have when launching any video. Console was opened on FT startup:
![image](https://user-images.githubusercontent.com/28269333/226748340-8905e5b8-dc6c-441c-85b1-8d4f9cb04e11.png)


This is startup only:
![image](https://user-images.githubusercontent.com/28269333/226748509-024728a8-5d72-49bf-91e1-07ffa8f626ba.png)

**Comment by maplepy** [comment]
[freetube.log](https://github.com/FreeTubeApp/FreeTube/files/11034068/freetube.log)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> This is the new part when I launch a video: ![image](https://user-images.githubusercontent.com/28269333/226748657-efe2592b-e2a1-47d7-990e-7e4b22ac8778.png)

The last one is the error we need. The log is very usefull but a bit hard to read, could u open the error by clicking on the at the arrow right at the beginning of the error.

**Comment by maplepy** [comment]
![image](https://user-images.githubusercontent.com/28269333/226946069-ebce0e7e-91dc-4d8a-acaa-093dc2f72ea5.png)

When loading a video, I know have more errors somehow
This is the full error:
![image](https://user-images.githubusercontent.com/28269333/226946392-7dbcb2da-3860-4c89-839c-440bb8738e1f.png)
![image](https://user-images.githubusercontent.com/28269333/226946468-6b628d31-2588-4b38-aff6-6349d63158e4.png)
![image](https://user-images.githubusercontent.com/28269333/226946546-788ceb4f-9365-424e-a82a-a6f7c5064a4c.png)

**Comment by maplepy** [comment]
Alright great ! Thanks to you :)
